portfolio/src/main/webapp/script.js

```diff
@@ -63,3 +64,21 @@ const addContentToDOM = content => {
 
   quoteContainer.innerText = content;
 };
+
+/** 
+ * Prints a random message to the DOM using JSON sent from DataServlet.java.
+ */
+const fetchMessageUsingJSON = () => {
+  fetch('/data').then(response => response.json()).then( messagesObj => {
```

Comment:
nit: space.
+askewc@: is there any auto-lint or auto-formatter?

Reply:
I'm confused. Where do you want to see the space?

Reply:
After the second "then("

Reply:
In the examples they gave us it was in all one line, but I'll change it

Reply:
const fetchMessageUsingJSON = () => {
  fetch('/data').then(response => response.json()).then( 
    messagesObj => {
    // messagesObj is an object, not a string, so we have to
    // reference its fields to create HTML content
    
    // Get random message from the the "messages" field
    // of messagesObj
    const messagesSize = messagesObj.messages.length;
    const message = messagesObj.messages[Math.floor(Math.random() * messagesSize)];

    // Add message to the page
    factContainer.innerText = message;
  });
};

Reply:
`const fetchMessageUsingJSON = () => {
  fetch('/data').then(response => response.json()).then( 
    messagesObj => {
    // messagesObj is an object, not a string, so we have to
    // reference its fields to create HTML content
    
    // Get random message from the the "messages" field
    // of messagesObj
    const messagesSize = messagesObj.messages.length;
    const message = messagesObj.messages[Math.floor(Math.random() * messagesSize)];

    // Add message to the page
    factContainer.innerText = message;
  });
};
`

Is this what you want it to look like?

Reply:
`const fetchMessageUsingJSON = () => {
  fetch('/data').then(response => response.json()).then( 
    messagesObj => {
    // messagesObj is an object, not a string, so we have to
    // reference its fields to create HTML content
    
    // Get random message from the the "messages" field
    // of messagesObj
    const messagesSize = messagesObj.messages.length;
    const message = messagesObj.messages[Math.floor(Math.random() * messagesSize)];

    // Add message to the page
    factContainer.innerText = message;
  });
};
`

Reply:
I tried posting it here to see how you want it but I keep getting this weird formatting

Reply:
use three backticks for blocks of code, I bet